CHANGELOG.md

```
@@ -84,5 +84,6 @@ scoping strategies (global / per-page / per-content) and three timing strategies
 (dynamic / scheduler / hybrid), a backend module, CLI commands and a Reports module
 entry, for TYPO3 v12.4 and v13 on PHP 8.1-8.3.
 
-[Unreleased]: https://github.com/netresearch/t3x-nr-temporal-cache/compare/v0.9.0-alpha1...main
+[Unreleased]: https://github.com/netresearch/t3x-nr-temporal-cache/compare/v0.9.0...main
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)